lib/unsplash/photo.rb

```diff
@@ -83,13 +85,15 @@ def random(count: nil, collections: nil, featured: nil, user: nil, query: nil, o
       # @param page  [Integer] Which page of search results to return.
       # @param per_page [Integer] The number of users search result per page. (default: 10, maximum: 30)
       # @param orientation [String] Filter by orientation of the photo. Valid values are landscape, portrait, and squarish.
+      # @param content_filter [String] Limit results by content_filter to avoid content that may be unsuitable for younger audiences. Valid values are low, high. Defaults to low.
       # @return [SearchResult] a list of +Unsplash::Photo+ objects.
-      def search(query, page = 1, per_page = 10, orientation = nil)
+      def search(query, page = 1, per_page = 10, orientation = nil, content_filter = 'low')
```

Comment:
Sorry didn't notice this before, prefer double quotes for strings.

Reply:
@internethostage - Changes done as per review.